### ISSUE TITLE
Visual focus indicator does not move to the help control.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#128]
User experience:
After pressing enter key on the "api details" button, if the visual focus indicator does not move to the open dialog box help control, then the user who depends upon keyboard for navigation might get affected hence has to use extra tab key to reach the help button.

Note: User credentials should NOT be included in the bug.

Repro steps:

1. Open Azure portal with valid credentials and create a API Management.
2. Click Developer Portal button to navigate to the portal.
3. Wait portal to load in design mode
4. Navigate to the Left nav and select Pages button.
5. Navigate to the Pages window. Select "APIs: List" option.
6. Navigate to the any editable control details and select Add widget.
7. Navigate to the List of API and select it.
8. Navigate to the various controls of the "List of API" dialog box.
9. Navigate to the api details button and press enter key.
10. Verify, whether the visual focus indicator move to the first interactive element i.e help control or not.

Actual Result:

After pressing enter key on the api details button, then the visual focus indicator does not move to the help control. Instead visual focus indicator move to the search page control.
 

Expected Result:

After pressing enter key on the api details button, then the visual focus indicator should move to the help control.

Note: 1. This issue also exist on the chrome browser.
          2. This issue also exist on web url. After pressing enter key on the web url visual focus does not move to the next interactive element.
 
MAS Reference:

MAS 3.2.1 - On Focus

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it      back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.

**Comment by azaslonov** [comment]
Fixed.
